src/opm/io/eclipse/OutputStream.cpp

```diff
@@ -396,6 +426,84 @@ namespace Opm { namespace EclIO { namespace OutputStream {
 
 }}}
 
+// =====================================================================
+
+Opm::EclIO::OutputStream::RFT::
+RFT(const ResultSet&    rset,
+    const Formatted&    fmt,
+    const OpenExisting& existing)
+{
+    const auto fname = outputFileName(rset, FileExtension::rft(fmt.set));
+
+    this->open(fname, fmt.set, existing.set);
+}
+
+Opm::EclIO::OutputStream::RFT::~RFT()
+{}
+
+Opm::EclIO::OutputStream::RFT::RFT(RFT&& rhs)
+    : stream_{ std::move(rhs.stream_) }
+{}
+
+Opm::EclIO::OutputStream::RFT&
+Opm::EclIO::OutputStream::RFT::operator=(RFT&& rhs)
+{
+    this->stream_ = std::move(rhs.stream_);
+
+    return *this;
+}
+
+void
+Opm::EclIO::OutputStream::RFT::
```

Comment:
Ehhhhh - this looks like a missing base class functionality?

Reply:
> this looks like a missing base class functionality?

Partly.  I don't want to support an unbounded overload set here, only those element types that make sense in RFT files.  I don't really know how I'd combine the templated nature of `EclOutput` with the virtual functions I'd need to implement this entirely in terms of a base class.